app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+<p style="color: green"><%= notice %></p>
+<div>
+<h1 class="header">Purchases</h1>
+<%= link_to "New purchase", new_category_purchase_path  ,class: "btn btn-info"%>
+</div>
+  <div id="purchases">
+        <%= render 'shared/purchase_table' do %>
+          <% @purchases.each do |purchase| %>
+            <%= render purchase %>
+          <% end %>
+        <% end %>
+      </div>
+      </div>
+
```

Comment:
- [ ] Same here, As per this [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow%20these%20design%20guidelines%2C%20including%3A), Please follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

- Colors.
- Typography: font face, size and weight.
- Layout: composition and space between elements.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/208918033-574a0aa5-17ae-4102-bf02-4b399b85236d.png) |![image](https://user-images.githubusercontent.com/68796232/208917941-25100c03-da28-4e6a-b0bf-774fd343d91a.png) |